composer.json

```diff
@@ -29,6 +29,12 @@
         "process-timeout": 1800
     },
     "scripts": {
+         "post-install-cmd": [
+            "php -r \"if(!file_exists('config/console-local.php')){copy('config/templates/console-local.php', 'config/console-local.php');}\"",
+            "php -r \"if(!file_exists('config/web-local.php')){copy('config/templates/web-local.php', 'config/web-local.php');}\"",
+            "php -r \"if(!file_exists('config/db-local.php')){copy('config/templates/db-local.php', 'config/db-local.php');}\"",
+            "yii\\composer\\Installer::postCreateProject"
+        ],
```

Comment:
This is only temporary solution. This could be simplified after merge https://github.com/yiisoft/yii2-composer/pull/10 and https://github.com/yiisoft/yii2-composer/pull/12